src/components/controls.tsx

```diff
@@ -11,6 +11,8 @@ const StyledControls = styled.div`
   flex-direction: column;
   justify-content: flex-start;
   align-items: flex-start;
+  background-color: #F0F0F0;
```

Comment:
We probably need a central location for commonly used colors from the UI spec.  I'm open to ideas.

Reply:
Seems like a good idea to have a file specific for those style-related vars at a minimum.

We could also look into pulling those vars in via `<ThemeProvider>` but that may be overkill for what we are doing here.

Reply:
I'll look at pulling all of the colors out of the spec and putting them in one module in a separate PR.  But I think we both agree that it would be useful.